tests/test_conftest.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+"""Tests for fixtures in the ``conftest.py``."""
+import pytest
+
+from aiida.common.exceptions import MissingEntryPointError
+from aiida.plugins.entry_point import EntryPoint, get_entry_point, load_entry_point
+
+ENTRY_POINT_GROUP = 'aiida.calculations.importers'
+
+
+def test_entry_points_add_invalid(entry_points):
+    """Test the :meth:`EntryPointManager.add` method."""
+    with pytest.raises(TypeError, match='`entry_point_string` should be a string when defined.'):
+        entry_points.add('some.module:SomeClass', [])
```

Comment:
After adding to the entry point, should it be cleaned after the test?

Reply:
In this case it wouldn't even be added, because the add call is invalid and an exception is thrown. But even if it would have been valid and an entry point would be added, the cleanup is done automatically by the fixture. The actual entrypoints are monkeypatched by the fixture, so by the time the fixture goes out of scope (i.e. after the tests) the original entry points are simply restored.